host/src/components/Responses/ResponsesGraph.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { Typography, makeStyles } from '@material-ui/core';
+import { VictoryChart, VictoryAxis, VictoryBar, VictoryContainer } from 'victory';
+import CustomTick from './CustomTick';
+import CustomLabel from './CustomLabel';
+import CustomBar from './CustomBar';
+
+export default function ResponsesGraph ({ 
+  data, 
+  questionChoices, 
+  statePosition, 
+  graphClickInfo, 
+  setGraphClickInfo 
+}) {
+  const classes = useStyles();
+  const [boundingRect, setBoundingRect] = useState({ width: 0, height: 0 });
+  const graphRef = useRef(null);
+  const barThickness = 18;
+  const barThicknessZero = 26;
+  const xSmallPadding = 4;
+  const smallPadding = 8;
+  const mediumPadding = 16;
+  const mediumLargePadding = 20;
+  const largePadding = 24;
+  const labelOffset = 3;
+  const noResponseLabel = '–';
+  // victory applies a default of 50px to the VictoryChart component
+  // we intentionally set this so that we can reference it programmatically throughout the chart
+  const defaultVictoryPadding = 50;
+  const correctChoiceIndex = questionChoices.findIndex(({ isAnswer }) => isAnswer) + 1;
+  const largestAnswerCount = Math.max(...data.map(response => response.answerCount));
+  
+  const calculateRoundedTicks = () => {
+    const maxAnswerCount = Math.max(...data.map(({ answerCount }) => answerCount));
+    const tickInterval = 5;
+    const tickCount = Math.ceil(maxAnswerCount / tickInterval);
+    return Array.from({ length: tickCount + 1 }, (_, index) => index * tickInterval);
+  };
+
+  useEffect(() => {
```

Comment:
This is required to handle resizing of the screen. This pushes Victory to regenerate the svgs for the chart on resize.